### PR TITLE
Add licensing/attribution list

### PR DESCRIPTION
**Pull request by connor4312**
Legal stuff

### CODE REVIEW

**Comment by JamyDev** [comment]
lgtm

**Comment by ProbablePrime** [comment]
yes

**Comment by JamyDev** [comment]
Only JS modules in this list?

**Comment by connor4312** [comment]
nope this is everything

**Comment by JamyDev** [comment]
Ok!

File: src/legal/attributions.pug [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+block navbar
+    .navbar-wrapper
+        .container
+            nav.navbar.navbar-inverse: +navbar()
+            h1 Attributions &amp; Licenses
+
+block content
+    .container
+        .page-header
+            p Here at Beam we use a significant amount of code from the open source community, and we try to #[a(href='https://github.com/WatchBeam') give back] when we can too! Listed below are licenses for many of the modules and submodules used in various places across the Beam platform.
```
Comment (2Cubed) [inline src/legal/attributions.pug]: :heart:

**Pull request merged**